### DISCUSSION TITLE
Welcome to arrows.app Discussions!

### DISCUSSION THREAD

**Discussion by akollegger** [neo4j-labs/arrows.app#85] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by niblux** [comment]
Hello,

Does Arrows currently offer an API or a feature to export diagrams as SVG code? I need to export our diagrams as SVG to render them using D3 in an Angular application. Any guidance on this would be greatly appreciated.

Thank you.
